RAW_RELEASE_NOTES.md

```diff
@@ -8,3 +8,5 @@ will make it substantially easier for the releaser to "linkify" all of the relea
 final version.
 
 ## 1.6.0
+
+* Added `version info` string to the `/clusters` admin endpoint.
```

Comment:
This will likely make it into 1.5.0. Remove from here and add to release notes in the other repo.